.github/workflows/CI.yaml

```diff
@@ -30,16 +30,14 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Checkout Code
         uses: actions/checkout@v3
-      - name: Install poetry
-        run: pip install poetry
-      - name: Install repo with poetry (full deps)
-        if: matrix.python-version != '3.9'
-        run: poetry install --no-interaction --no-ansi --all-extras
-      - name: Install repo with poetry (min deps)
-        if: matrix.python-version == '3.9'
-        run: poetry install --no-interaction --no-ansi --extras test
+      - name: Install QCElemental (full deps)
+        if: (!(matrix.python-version == '3.8' || matrix.python-version == '3.10'))
```

Comment:
Nit: I had to spend more time than I'd care to admin looking at this step and the next one to figure out why they both exist (my mind was thinking that they both had the same conditional).

I don't think that this needs a change, but some options that may have saved time for a person who thinks like me :) :

*   Comments to indicate that one clause was for 3.8 and 3.10 and another for other python versions
*   Convert `!(matrix.python-version == '3.8' || matrix.python-version == '3.10')` to `matrix.python-version != '3.8' && matrix.python-version != '3.10'`
*   Convert `!(matrix.python-version == '3.8' || matrix.python-version == '3.10')` to `matrix.python-version == '3.9' || matrix.python-version == '3.11' || matrix.python-version == '3.12' || matrix.python-version == '3.13'` (I don't like that this option needs to be maintained as python versions come and go)